src/Features/CSharp/Portable/ConvertProgram/ConvertProgramTransform_ProgramMain.cs

```diff
@@ -45,9 +45,13 @@ public static async Task<Document> ConvertToProgramMainAsync(Document document,
                         Contract.ThrowIfNull(newRoot);
 
                         var firstGlobalStatement = newRoot.Members.OfType<GlobalStatementSyntax>().Single();
+                        var firstGlobalStatementLeadingTrivia = firstGlobalStatement.GetLeadingTrivia();
```

Comment:
Can we use IBannerFacts instead?

Reply:
Searched for `IBannerFacts` in the solution, but didn't find anything...

Reply:
Maybe IHeaderFacts?